strawberry_django_plus/relay.py

```diff
@@ -329,10 +329,14 @@ class Node(abc.ABC):
 
     All types that are relay ready should inherit from this interface and
     implement the following methods.
+    Attributes:
```

Comment:
Add a visual separation here

```suggestion

    Attributes:
```